app/src/components/common/form/common_styled/index.tsx

```diff
@@ -1,4 +1,9 @@
-import { css } from 'styled-components'
+import styled, { css } from 'styled-components'
+
+interface StatefulRadioButton {
```

Comment:
Instead of placing these components in `/form/common_styled`, they'd be better suited in `/market/common/common_styled` since they're market related

Reply:
@KadenZipfel submitted changes! also did you see comment i made on issue about code?

Reply:
https://github.com/protofire/omen-exchange/issues/1344#issuecomment-729126137